core/src/test/java/org/elasticsearch/rest/action/support/RestTableTests.java

```diff
@@ -70,6 +95,65 @@ public void testThatDisplayHeadersAreNotAddedTwice() throws Exception {
         assertThat(headerNames, not(hasItem("unmatched")));
     }
 
+    public void testThatWeUseTheAcceptHeaderJson() throws Exception {
+        assertResponse(Collections.singletonMap(ACCEPT, APPLICATION_JSON),
+                APPLICATION_JSON,
+                JSON_TABLE_BODY);
+    }
+
+    public void testThatWeUseTheAcceptHeaderYaml() throws Exception {
+        assertResponse(Collections.singletonMap(ACCEPT, APPLICATION_YAML),
+                APPLICATION_YAML,
+                YAML_TABLE_BODY);
+    }
+
+    public void testThatWeUseTheAcceptHeaderSmile() throws Exception {
+        assertResponseContentType(Collections.singletonMap(ACCEPT, APPLICATION_SMILE),
+                APPLICATION_SMILE);
+    }
+
+    public void testThatWeUseTheAcceptHeaderCbor() throws Exception {
+        assertResponseContentType(Collections.singletonMap(ACCEPT, APPLICATION_CBOR),
+                APPLICATION_CBOR);
+    }
+
+    public void testThatWeUseTheAcceptHeaderText() throws Exception {
+        assertResponse(Collections.singletonMap(ACCEPT, TEXT_PLAIN),
+                TEXT_PLAIN,
+                TEXT_TABLE_BODY);
+    }
+
+    public void testIgnoreContentType() throws Exception {
+        assertResponse(Collections.singletonMap(CONTENT_TYPE, APPLICATION_JSON),
+                TEXT_PLAIN,
+                TEXT_TABLE_BODY);
+    }
+
+    private RestResponse assertResponseContentType(Map<String, String> headers, String mediaType) throws Exception {
+        FakeRestRequest requestWithAcceptHeader = new FakeRestRequest(headers, Collections.emptyMap());
```

Comment:
This does not compile; `FakeRestRequest` does not have a constructor with two arguments. This is a recent change and I think you just missed it when you rebased.